src/arrayMethodSort.js

```diff
@@ -4,8 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareString(a, b) {
+    const firstString = String(a);
+    const secString = String(b);
```

Comment:
You can name it full: like secondString. It's not very long name :) Or better secondStr

Reply:
Got it, thanks) don't like long names and it's difficult to force yourself to write them fully, but I'm trying.